config/vmdb.tmpl.yml

```diff
@@ -344,6 +344,8 @@ workers:
           :poll_method: :escalate
         :ems_metrics_collector_worker_kubernetes:
           :poll_method: :escalate
+          :hawkular_port: 5000
```

Comment:
Right, this is a limitation in ManageIQ right now.  We don't have a way to collect a user-supplied port for services other than the main API yet.